functions/src/tests/index.test.ts

```diff
@@ -77,12 +77,3 @@ describe("GET /shareLinksByUrl", () => {
     }
   );
 });
-
-describe("GET /screenshot", () => {
```

Comment:
I'm not really sure how to test this with these changes 😕 maybe we could point to the example share page in `act-now-template` but it feels eh to rely on another repo for a unit test

Reply:
Hrm, yeah.  I guess we could make the class names customizable as url parameters or something.  But I'm okay with dropping the test for now.